exercises/practice/gigasecond/.docs/instructions.append.md

```diff
@@ -0,0 +1,3 @@
+# Hints
```

Comment:
Just in case you didn't know: you can also use `hints.md` in practice exercises. It is less visible than putting it here, as this content will get appended to the README.md and the hints are a separate file.